create_database/pubtator-to-postgres.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Inserting Pubtator into a Database
+
+# This notebook is designed to load each [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/) abstract. It uses our own [pubtator repository](https://github.com/greenelab/pubtator) to convert each abstract into the appropriate format (xml) to load into a postgres database. Run the pubtator scripts before running this notebook, so the pubtator xml file can be constructed. Once constructed this notebook is designed to parse the data into a database. After loading each abstract, candidate extraction is performed for each relationship type.
+
+# ## MUST RUN AT THE START OF EVERYTHING
```

Comment:
I am having this same issue currently so I'm not sure the best solution for this.  But since you only have to run this top set of scripts once.  Does it make more sense to have it in its own separate nb?  Or have a wrapper script to call these set of scripts and keep track of if was already run based on the output?

Reply:
Well it depends on the design. If the imports for each notebook are the same, then it makes sense to move the code into a separate script and then just have the notebooks call that import script. In this situation this is the only time I need to have the following code as future work won't even use this notebook. I just have it here for project completion sake.